agency/README.md

```diff
@@ -263,6 +272,35 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `gps.hdop`     | Float          | Required if Available | Horizontal GPS or GNSS accuracy value (see [hdop](https://support.esri.com/en/other-resources/gis-dictionary/term/358112bd-b61c-4081-9679-4fca9e3eb926)) |
 | `gps.satellites` | Integer      | Required if Available | Number of GPS or GNSS satellites
 | `charge`       | Float          | Required if Applicable | Percent battery charge of vehicle, expressed between 0 and 1 |
+| `stop_id`      | UUID           | Required if Applicable | Stop that the vehicle is currently located at. Only applicable for _docked_ Micromobility. See [Stops](../general-information.md#stops) |
+
+## Stops
+
+The `/stops` endpoint allows an agency to register Stops.
```

Comment:
When I first began working on the concept of Stops, it was initially designed such that a city would define the Stops that providers could use (e.g. could be used for designating virtual drop-off zones like the Venice Beach Special Operations Zone). That being said, I think you're right that we need a mechanism for providers registering stops with an agency via the Agency API as well I think. I'll reword so that it it something like:
```suggestion
The `/stops` endpoint allows an agency to register city-managed Stops, or a provider to register self-managed Stops.
```

Reply:
We probably ought to also extend it such that Stops in Agency have some encoding for who manages the stop. Perhaps just a `provider_id`?

Reply:
> When I first began working on the concept of Stops, it was initially designed such that a city would define the Stops that providers could use

👍 Cities don't need to use the API to create stops since they have access to the underlying implementation but it can't do any harm to mention they can.

> We probably ought to also extend it such that Stops in Agency have some encoding for who manages the stop. Perhaps just a `provider_id`?

I agree we should add a `provider_id`, also in case a provider manages a stop for another provider.

Reply:
@avatarneil it looks like the wording update 

> The `/stops` endpoint allows an agency to register city-managed Stops, or a provider to register self-managed Stops.

got missed in the merging process, could you make a quick update including it?

Reply:
Resolved here: https://github.com/openmobilityfoundation/mobility-data-specification/pull/539/commits/983a99fbb84d6d2ea805bcb82676a63bb431a605